backend/apps/github/models/issue.py

```diff
@@ -70,6 +70,13 @@ class Meta:
     )
 
     # M2Ms.
+    interested_users = models.ManyToManyField(
```

Comment:
It's not that simple. 
Let's create a through table for program/user/issue combinations as within time issue can be involved in multiple different programs.